notebook_diff/test.ipynb

```diff
@@ -53,33 +53,21 @@
        "    <tr>\n",
```

Comment:
<p>This is wrong</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pettinato/poc_work/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>